specification/portal/resource-manager/Microsoft.Portal/preview/2019-01-01-preview/tenantConfiguration.json

```diff
@@ -0,0 +1,204 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2019-01-01-preview",
```

Comment:
>2019-01-01-preview [](start = 16, length = 18)

This API version is older than a year and will be automatically treated as "stable". You should consider choosing something more recent for this API.

Take a look at the Versioning parts of [https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md#versioning](https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md#versioning).

Reply:
2019-01-01-preview is the latest version at this moment - that's why I pick it
Though soon there will be newer version - Himanshu is preparing some changes which will require version update.